### PR TITLE
Tutor feed api

### PR DESCRIPTION
**Pull request by GoldenJ19**

### CODE REVIEW

File: src/pages/api/tutor/subjects/[tutorId].ts
Comment (NeoThundr) [inline src/pages/api/tutor/subjects/[tutorId].ts:1]: Can you rename this folder from "subjects" to something like tut_subs or something to make it more different than the other folder called "subject"

File: src/pages/api/tutor/index.ts [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
 export default async function handler(
     req: NextApiRequest,
-    res: NextApiResponse<tutor[] | string>
+    res: NextApiResponse<TutorWithSubjects[] | string>
 ) {
     switch (req.method) {
         case 'GET':
-            const allTutors = await prisma.tutor.findMany();
+            const allTutors = await prisma.tutor.findMany({
+                include: {
```
Comment (NeoThundr) [inline src/pages/api/tutor/index.ts:18]: Why is it "include" and not "Where".

**Review by NeoThundr — COMMENTED** [review]
I added a few comments. Change a folder name from "subjects" to something like tutors_subjects to seperate it from the subject folder

File: src/pages/api/tutor/subjects/[tutorId].ts
Comment (micahkatz) [inline src/pages/api/tutor/subjects/[tutorId].ts:1]: You can delete this folder

**Review by micahkatz — APPROVED** [review]

File: src/pages/api/tutor/index.ts (hunk 1 above)
Comment (GoldenJ19) [reply src/pages/api/tutor/index.ts:18]: answered on discord

File: src/pages/api/tutor/subjects/[tutorId].ts
Comment (GoldenJ19) [reply src/pages/api/tutor/subjects/[tutorId].ts:1]: talked over, will leave for now for later editing.

**Pull request merged**